### DISCUSSION TITLE
Query regarding Jetty CVE-2024-22201

### DISCUSSION THREAD

**Discussion by sandeepnkulkarni** [dropwizard/dropwizard#8463] [Development]
Recently Jetty vulnerability  CVE-2024-22201 was discovered. More details: https://github.com/advisories/GHSA-rggv-cv7r-mw98

Security scanner have detected it in latest version of Dropwizard 4.0.6 as well because it is using affected version.

Any plans to release a new version of Dropwizard with patched Jetty version?

**Comment by joschi** [comment]
Thanks for bringing this up! ❤️ 

The issue will be addressed in Dropwizard 3.0.7 and 4.0.7 once the CI pipeline completed and the artifacts were synced to Maven Central.

- https://github.com/dropwizard/dropwizard/releases/tag/v3.0.7
- https://github.com/dropwizard/dropwizard/releases/tag/v4.0.7

**Reply by sandeepnkulkarni** [reply]
Thanks a lot for quick response and release as well.
